docs/running-calyx/fud2/rhai-api.md

```diff
@@ -0,0 +1,152 @@
+# Rhai API
+
```

Comment:
Maybe it would be handy to have a few sentences of context here? For example:

* reminder of what fud2 is, with a link to the main page
* this is documentation for people who want to extend the functionality of fud2, not for people who just want to use it
* what is Rhai?
* what is the difference between the two "levels"
* a reminder of what you need to do to get started: i.e., create a file called `something.rhai` in `calyx/fud2/scripts`, or put it somewhere else and use the `plugins` key in fud2's configuration to point to your script